src/modules/70-pipeline/pages/webhooks/WebhookEvents/WebhookEvents.tsx

```diff
@@ -199,6 +243,9 @@ export default function WebhookEvents(): JSX.Element {
             <Text color={Color.GREY_800} iconProps={{ size: 14 }}>
               {getString('total')}: {data?.pagination?.total}
             </Text>
+            <span className={css.reload}>
+              <Icon name="command-rollback" onClick={() => refetch()} />
```

Comment:
This will be completely invisible as a clickable element to a screenreader or other assistive technologies. Instead use a `Button` element.
```suggestion
              <Button onClick={() => refetch()} icon="command-rollback" variation={ButtonVariation.ICON} />
```